File: openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/MailStatsTypeVolume.scala

package org.openapitools.client.model


case class MailStatsTypeVolume (
    /* Message counts keyed by destination (envelope `to`) email address. */
    _to: Option[Map[String, Integer]],
    /* Message counts keyed by sender (envelope `from`) email address. */
    _from: Option[Map[String, Integer]],
    /* Message counts keyed by originating client IP address. */
    _ip: Option[Map[String, Integer]]
)
object MailStatsTypeVolume {
    def toStringBody(var_to: Object, var_from: Object, var_ip: Object) =
        s"""
        | {
        | "to":$var_to,"from":$var_from,"ip":$var_ip
        | }
        """.stripMargin
}

File: openapi-client/javascript-closure-angular/API/Client/MailStatsTypeVolume.js
goog.provide('API.Client.MailStatsType_volume');

/**
 * Top-500 breakdown of message counts grouped by source IP, destination address, and sender address within the selected `time` window.
 * @record
 */
API.Client.MailStatsTypeVolume = function() {}

/**
 * Message counts keyed by destination (envelope `to`) email address.
 * @type {!Object<!string, number>}
 * @export
 */
API.Client.MailStatsTypeVolume.prototype.to;

/**
 * Message counts keyed by sender (envelope `from`) email address.
 * @type {!Object<!string, number>}
 * @export
 */
API.Client.MailStatsTypeVolume.prototype.from;

/**
 * Message counts keyed by originating client IP address.
 * @type {!Object<!string, number>}
 * @export
 */
API.Client.MailStatsTypeVolume.prototype.ip;

